src/t2iapi/metric/metric_requests.proto

```diff
@@ -57,4 +57,13 @@ message SetMetricStatusRequest {
     string handle = 1; // handle of the metric for which the status shall be changed
     MetricStatus status = 2; // the measurement, calculation or setting status the metric should be set to, to trigger
     // a certain AbstractMetricState/@ActivationState
-}
\ No newline at end of file
+}
+
+/*
+Request to set the ActivationState of the metric with the given handle to the given ActivationState value.
+ */
+message SetActivationStateRequest {
+    string handle = 1;  // descriptor handle of the descriptor for which the ActivationState shall be set
+                        // in the corresponding state element
+    ActivationState activation_state = 2;  // value to set as the ActivationState value
```

Comment:
what about "activation_state.ComponentActivation" instead of ActivationState?

Reply:
done